### PR TITLE
*: fix descriptions about "explicit"

### PR DESCRIPTION
**Pull request by lilin90**
@CaitinChen PTAL
Via: https://github.com/pingcap/docs-cn/pull/1108

Note: Currently, the docs repo lacks the "sql/date-and-time-types.md" file, so the update in this file is not involved in this PR. That file will be added in another PR.

### CODE REVIEW

File: sql/util.md [hunk 1]
```diff
@@ -96,4 +96,4 @@ If the `dot` program is not installed on your computer, copy the result to [this
 USE db_name
 ```
 
-The `USE` statement is used to switch the default database. If the table in SQL statements does not display the specified database, then use the default database.
+The `USE` statement is used to switch the default database. If the table in this SQL statement does not have an explicitly specified database, then the default database is used.
```
Comment (yikeke) [inline sql/util.md]: How about "If the table in this SQL statement does not correspond to an explicitly specified database, then the default database is used."? (I thought the database contains the table. It's not that the table has the database?)

File: v1.0/sql/util.md [hunk 2]
```diff
@@ -93,4 +93,4 @@ If the `dot` program is not installed on your computer, copy the result to [this
 USE db_name
 ```
 
-The `USE` statement is used to switch the default database. If the table in SQL statements does not display the specified database, then use the default database.
+The `USE` statement is used to switch the default database. If the table in this SQL statement does not have an explicitly specified database, then the default database is used.
```
Comment (yikeke) [inline v1.0/sql/util.md]: ditto

File: v2.0/sql/util.md [hunk 3]
```diff
@@ -96,4 +96,4 @@ If the `dot` program is not installed on your computer, copy the result to [this
 USE db_name
 ```
 
-The `USE` statement is used to switch the default database. If the table in SQL statements does not display the specified database, then use the default database.
+The `USE` statement is used to switch the default database. If the table in this SQL statement does not have an explicitly specified database, then the default database is used.
```
Comment (yikeke) [inline v2.0/sql/util.md]: ditto

File: v2.1/sql/util.md [hunk 4]
```diff
@@ -96,4 +96,4 @@ If the `dot` program is not installed on your computer, copy the result to [this
 USE db_name
 ```
 
-The `USE` statement is used to switch the default database. If the table in SQL statements does not display the specified database, then use the default database.
+The `USE` statement is used to switch the default database. If the table in this SQL statement does not have an explicitly specified database, then the default database is used.
```
Comment (yikeke) [inline v2.1/sql/util.md]: ditto

**Comment by CaitinChen** [comment]
Agree with @yikeke 
Rest LGTM

**Pull request merged**